opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "1.2"
+maintainer: "thomas@gazagnaire.org"
+authors: [ "Louis Gesbert" "Thomas Gazagnaire" ]
+license: "ISC"
+build: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+remove: [
+  ["ocamlfind" "remove" "lazy-trie"]
+]
+build-doc: [ "ocaml" "setup.ml" "-doc" ]
+depends: [
+  "ocamlfind"
+  "sexplib"
+  "type_conv"
+  "camlp4"
```

Comment:
`type_conv` and `campl4` likely need to be replaced with the appropriate ppx here.

Reply:
Right, fixed.